src/shared/views/plugins/Model3d/Model3d.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Model3d = (props) => {
+  return (
+    <div>
+      <iframe width="640" height="480" src={props.url} frameBorder="0" allowFullScreen />
```

Comment:
Need to change width to 100%. Height is also a bit much for the iphone 4, but we can deal with that when we style.